compile

```diff
@@ -1 +0,0 @@
-/usr/share/automake-1.15/compile
```

Comment:
If you do this, you might also want to update the build instructions, otherwise compilation fails for me with
```
configure.ac:5: error: required file './compile' not found#
configure.ac:5:   'automake --add-missing' can install 'compile'
```
when executing `make` right after `./configure`